testing_json_object.rb

```diff
@@ -0,0 +1,39 @@
+require 'open-uri'
+require 'json'
+require 'faker'
+
+def parse_dogs_json
+  @dog_data = JSON.parse(IO.read('dogsData.json'))
+end
+
+
+# create dog records in our database
+def seed_dogs(data)
+  # data = @dog_data["collection1"]
+
+  i = 0
+  59.times do
```

Comment:
one json is parsed you get standard ruby objects so things like `.each` or `.each_with_index` work. There is no need to use `i=0` counter variables.